.gitlab-ci.yml

```diff
@@ -53,6 +53,8 @@ stages:
     paths:
       - 'packages/*/build*'
       - 'apps/*/build*'
+  variables:
+    NODE_ENV: 'production'
```

Comment:
will this not set the `NODE_ENV` as `production` for all environments?

Reply:
Isn't this something that we want? Usually `development` is only set for local developer machines.
We want dev/PR environments to be as close to the prod environment as possible, right?

I did experience differences between development/production builds in the past - and feature flag would be one example where there are also differences.

Reply:
the only thing we check with it is the Feature Flags and to register a service worker. The default behavior for feature flags, should be On on Dev and PR's and off on Prod

Reply:
We do want to keep feature flags enabled by default on development. Perhaps we have overloaded NODE_ENV

Reply:
@kmaid @jhonny-m how about we use REACT_APP_ENVIRONMENT to enable feature flags by default? It is only set to production in production.